README.md

```diff
@@ -191,6 +191,7 @@ helm upgrade amko-1598451370 oci://projects.registry.vmware.com/ako/helm-charts/
 | `configs.federation.memberClusters`            | member clusters on which federation should be done                                                                                    |  |
 | `configs.gslbLeaderController`                         | GSLB leader site URL                                                                                                     | Nil                                   |
 | `configs.tenant`                         | GSLB leader controller tenant                                                                                                   | `admin`                                    |
+| `configs.amkoUUID`                         | Configure AMKO with a unique UUID in standard form of xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx with hex-digits.Should be used to recover AMKO state if [GSLBConfig](docs/crds/gslbconfig.md) is deleted                                                                                                  | AMKO will self-generate if left empty                                    |
```

Comment:
should we re-word it as `Should be used to recover AMKO state if [GSLBConfig](docs/crds/gslbconfig.md) is deleted. AMKO will self-generate it if left empty` ?

Reply:
done